### DISCUSSION TITLE
Timecodes out of sync

### DISCUSSION THREAD

**Discussion by djkiuru** [openai/whisper#1024] [Q&A]
I have a file in Spanish that I have been trying to get subtitles (SRT) for a while. I tried all models, and the timecodes are not aligned, even fixing them with subtitle editing software doesn't seem to work as they are not delayed or sped up by a specific length of time, they are all different. Any ideas? Thanks in advanced!

**Comment by glangford** [comment]
This is a known problem area, see these discussions -
* https://github.com/openai/whisper/discussions/89
* https://github.com/openai/whisper/discussions/435
